plugins/keycloak-backend/README.md

```diff
@@ -1,46 +1,46 @@
+| `baseUrl`      | Location of the Keycloak server, such as `https://localhost:8443/auth`. Note that the newer versions of Keycloak omit the `/auth` context path. | ""            | Yes                                                  |
+| `realm`        | Realm to synchronize                                                                                                                            | `master`      | No                                                   |
+| `loginRealm`   | Realm used to authenticate                                                                                                                      | `master`      | No                                                   |
+| `username`     | Username to authenticate                                                                                                                        | ""            | Yes if using password based authentication           |
+| `password`     | Password to authenticate                                                                                                                        | ""            | Yes if using password based authentication           |
+| `clientId`     | Client Id to authenticate                                                                                                                       | ""            | Yes if using client credentials based authentication |
+| `clientSecret` | Client Secret to authenticate                                                                                                                   | ""            | Yes if using client credentials based authentication |
 
-When using client credentials, the access type must be set to `confidential` and service accounts must be enabled. The following roles are required to be added from the `realm-management` client role:
+When using client credentials, the access type must be set to `confidential` and service accounts must be enabled. You must add the following roles from the `realm-management` client role:
 
-- query-groups
-- query-users
-- view-users
+- `query-groups`
+- `query-users`
+- `view-users`
 
 ## Installation
 
-1. The package must be installed into the backend of Backstage. When not integrating with a published package, the repository must be cloned locally and added to Backstage as shown below:
+1. Install the Backstage package into the backend. When not integrating with a published package, clone the repository locally and add the Backstage as follows:
 
    ```shell
    # From your Backstage root directory
    yarn add --cwd packages/backend @janus-idp/backstage-plugin-keycloak-backend
    ```
 
-2. Next add the basic configuration to `app-config.yaml`:
+2. Add the following configuration to `app-config.yaml` file:
```

Comment:
Just a tip for numbering in markdown; if you use `1.` for each numbered item, they should automatically generate the correct numbers. It's easier to maintain in the long run because e.g. if you remove a step or add one you don't have to manually update the numbering.
Example: https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list-in-markdown-format

Reply:
Thank you @abrennan89. 
However, we'll soon be converting the markdown files to asciidoc. If needed I'll make the change.